src/helpers.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+use Weglot\Client\Client;
+use Weglot\Client\Endpoint\Languages;
+use Weglot\Client\Api\LanguageEntry;
+use Illuminate\Support\Facades\Request;
+
+if (! function_exists('currentLocale')) {
+    /**
+     * Check current locale, based on URI segments
+     * @return string
+     */
+    function currentLocale()
+    {
+        $segment = Request::segment(1);
+        if (in_array($segment, config('weglot-translate.destination_languages'))) {
+            return $segment;
+        }
+        return config('weglot-translate.original_language');
+    }
+}
+
+if (!function_exists('currentRequestLocalizedUrls')) {
+    /**
+     * Returns array with all possible URL for current Request
+     * @return array
+     */
+    function currentRequestLocalizedUrls()
```

Comment:
Prefix with **weglot**. The function seems necessary so it must absolutely exist even if there is a conflict